lib/logger.js

```diff
@@ -1,23 +1,41 @@
 'use strict';
 
 const winston = require('winston');
+const util = require('util');
 
-module.exports = (config) => {
-
+module.exports = config => {
   const loggingTransports = [];
   const exceptionTransports = [];
   const notProd = config.env !== 'production';
 
+  const customFormatter = options => {
+    // - Return string will be passed to logger.
+    // - Optionally, use options.colorize(options.level, <string>) to
+    //   colorize output based on the log level.
+    return options.timestamp() + ' ' +
+      options.sessionID ? `sessionID=${options.sessionID}` : '' +
```

Comment:
I wonder if this would be neater?
```js
return [
    options.timestamp(),
    options.sessionID(),
    // ...
].filter(Boolean).join(' ')
```

Reply:
Yep, this was copied over from RIL and imagine was setup most likely by a SO post by the looks of things. I wasn't sure to leave it in or not but went with the former to see if it got some thoughts. Will implement the above. Much nice.